addons/l10n_account_withholding/models/chart_template.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    def _setup_utility_bank_accounts(self, template_code, company, template_data):
+        """ We need to define a default account for use in community, while when enterprise is installed it's up to the user or l10n to define them. """
+        super()._setup_utility_bank_accounts(template_code, company, template_data)
+        if company.parent_id or template_code in {'ar_ri', 'ar_ex', 'ar_base'} or company.l10n_account_withholding_tax_base_account_id:
```

Comment:
Make a global variable on top, so we can be more explicit on future changes (I think Italy at least can benefit).

 ```py
WITHHOLDING_TEMPLATES = {'ar_ri', 'ar_ex', 'ar_base'}
```
```suggestion
        if company.parent_id or template_code in WITHHOLDING_TEMPLATES or company.l10n_account_withholding_tax_base_account_id:
```

Reply:
:thinking: The reason to exclude AR chart template is because in AR, they cannot really use this module; instead they'll use their own which defines their own tax base account.
I'm not sure how it could help IT to add more templates here